src/test/scala/com/wavesplatform/UtxPoolSpecification.scala

```diff
@@ -50,6 +51,22 @@ class UtxPoolSpecification extends FreeSpec
   } yield TransferTransaction.create(None, sender, recipient, amount, time.getTimestamp(), None, fee, Array.empty[Byte]).right.get)
     .label("transferTransaction")
 
+  private def onlyOneValidFromManyTransfers = (for {
+    (sender, senderBalance, state) <- stateGen
+    recipient <- accountGen
+    fee <- chooseNum(1, (senderBalance * 0.01).toLong)
+    offset <- chooseNum(1000L, 2000L)
+  } yield {
+    val time = new TestTime(System.currentTimeMillis())
+    val utx = new UtxPool(group, time, state, calculator, FunctionalitySettings.TESTNET, UtxSettings(10, 10.minutes))
+    val amountPart = (senderBalance - fee) / 2 - fee
+    val txs = Seq(
+      PaymentTransaction.create(sender, recipient, amountPart, fee, time.getTimestamp() + 1).right.get,
```

Comment:
Check out `TestTime`: it should emit unique timestamps from `getTimestamp`. If it does not, fix it there, not in this test. I also suggest using Transfer transaction.